lib/torrent.js

```diff
@@ -568,10 +575,18 @@ class Torrent extends EventEmitter {
           cb(null)
         })
       })
-    }), FILESYSTEM_CONCURRENCY, err => {
-      if (err) return this._destroy(err)
-      this._debug('done verifying')
-      this._onStore()
+    }), FILESYSTEM_CONCURRENCY, cb)
+  }
+
+  rescanFiles (cb) {
+    if (!cb) cb = noop
+
+    // already checks for torrent destroyed
```

Comment:
This still needs a check for torrent destroyed that throws an error synchronously when the torrent is destroyed, i.e. `if (this.destroyed) throw new Error('torrent is destroyed')`. We do this on all public methods. This makes it easier to catch errors where the called doesn't realize the torrent is destroyed.

Reply:
Good to know. Fixed.